service/src/main/java/uk/gov/hmcts/reform/fpl/model/CaseData.java

```diff
@@ -172,7 +173,9 @@
     public final Document thresholdDocument;
     @JsonProperty("documents_socialWorkEvidenceTemplate_document")
     @Valid
-    private final Document socialWorkEvidenceTemplateDocument;
+    public final Document socialWorkEvidenceTemplateDocument;
+    @Valid
+    public final CourtBundle courtBundle;
```

Comment:
Hmm, not sure wether we need this validated.

Reply:
You are right. I will remove it.